qiskit_experiments/framework/base_experiment.py

```diff
@@ -348,7 +356,36 @@ def _run_jobs(self, circuits: List[QuantumCircuit], **run_options) -> List[Job]:
             job_circuits = [circuits]
 
         # Run jobs
-        jobs = [self.backend.run(circs, **run_options) for circs in job_circuits]
+        if self._use_sampler:
+            sampler = Sampler(self.backend)
```

Comment:
We can only support job execution mode when we instantiate sampler with backend.